calculator/src/App.js

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import Button from './components/Button';
+import Result from './components/Result';
+import './App.css';
+
+class App extends React.Component {
+  constructor(){
+    super();
+    this.state = { result: "" }
+  }
+
+  onClick = (symbol) => {
+    if(symbol === "="){
+      this.calculate()
+    } else if(symbol === "AC"){
+      this.reset()
+    } else if(symbol === "sign"){
+      this.signChange()
+    } else {
+      this.setState({ result: (this.state.result + symbol).toLocaleString("en") })
+    }
+  }
+
+  calculate = () => {
+    let resultCheck = ''
+    if(this.state.result.includes('--')){
+        resultCheck = this.state.result.replace('--','+')
```

Comment:
Having it being removed if they press the button again would be a more reliable option.